spec/controllers/v1/report_controller_spec.rb

```diff
@@ -3,109 +3,191 @@
 describe V1::ReportController do
   render_views
 
-  before :each do
-    request.env["HTTP_ACCEPT"] = 'application/json'
-  end
+  describe '#show' do
+    let(:assessment) { FactoryGirl.create(:assessment, :with_participants) }
+
+    context 'without user' do
+      before(:each) do
+        sign_out :user
+        get :show, assessment_id: 1, format: :json
```

Comment:
Even though we're going to fail this case regardless of what ID we pass through, it's better in terms of behavior and expressiveness to use the object that we intended here.  This way, if we see a corresponding successful test that uses the same assessment object, we can build more confidence in that this is testing the right thing.